packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/container.ts

```diff
@@ -93,6 +108,30 @@ export class Container extends ServiceExtension {
       containerProps = hookProvider.mutateContainerDefinition(containerProps);
     });
 
+    // If no observability extensions have been added to the service description, we can configure
+    // default awslogs log driver
+    if (!containerProps.logging) {
+      // Create a log group for the service, if one is not provided by the user
+      if (!this.logGroup) {
+        this.logGroup = new awslogs.LogGroup(this.scope, `${this.parentService.id}-logs`, {
+          logGroupName: `${this.parentService.id}-logs`,
+          removalPolicy: cdk.RemovalPolicy.DESTROY,
+          retention: awslogs.RetentionDays.ONE_MONTH,
+        });
+      }
+
+      containerProps = {
+        ...containerProps,
+        logging: new ecs.AwsLogDriver({
+          streamPrefix: this.parentService.id,
+          logGroup: this.logGroup,
+        }),
+      };
+    } else {
+      if (this.logGroup) {
+        throw Error(`A log configuration has already been specified for the service '${this.parentService.id}'. The default log group provided to the service container will not be used.`);
```

Comment:
The error message looks more like a warning to me. Maybe be more clear why it is an error like you cannot specify ... with ...

Reply:
Updated!